unique_toolkit/unique_toolkit/content/service.py

```diff
@@ -55,7 +64,9 @@ def async_search_content_chunks(
         search_string: str,
         search_type: ContentSearchType,
         limit: int,
-        scope_ids: Optional[list[str]],
+        reranker: Optional[RerankerConfig] = None,
+        fts_search_language: str = "english",
```

Comment:
As above, I would prefer reranker_config and search_language. Reuse default value as above. And make search_language optional and set to None.